components/FeedItemCard.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-hover>
+    <template v-slot="{ isHovering, props }">
+      <v-card v-bind="props">
+        <v-img :src="src()" aspect-ratio="1">
+          <template #placeholder>
+            <div class="d-flex align-center justify-center fill-height">
+              <v-container fluid class="fill-height">
+                <v-row class="ma-0">
+                  <v-col
+                    cols="12"
+                    class="pending font-weight-md-thin scale-text text-center"
+                  >
+                    {{ id }}
+                  </v-col>
+                  <v-col cols="12">
+                    <v-progress-linear
+                      indeterminate
+                      color="black"
+                      background-color="transparent"
+                      height="1"
+                      bottom
+                    />
+                  </v-col>
+                </v-row>
+              </v-container>
+            </div>
+          </template>
+          <template #error>
```

Comment:
not sure what this `#error` slot is.  i don't think it's doing what you want.  to test it out, just pass in any string you know isn't a real transaction id into the `<FeedComponent :id="'fake-id'" />`